### PR TITLE
Docs: Fix logging env vars example

### PR DESCRIPTION
**Pull request by leomp12**
Add `GOTRUE_` prefix to prevent confusion, also add `GOTRUE_LOG_FILE` sample value.

> Should also add `GOTRUE_LOG_FILE` to _example.env_?

**- Summary**

Minor doc fix because I've tried with `LOG_FILE` and it took me a few minutes to realize that the log was empty :disappointed: 
Then I tried with `GOTRUE_LOG_FILE` and it works :smile: 

**- Description for the changelog**

docs(readme): fix logging env vars example

**- A picture of a cute animal (not mandatory but encouraged)**

12670095_951550701593485_936418740864488038_n

### CODE REVIEW

**Comment by mraerino** [comment]
actually, `LOG_LEVEL` without the prefix works fine. it's explicitly handled specially in the config

**Comment by leomp12** [comment]
Hey @mraerino , but and `LOG_FILE`?
I've tried without prefix with not success, and Imo it's confuse to use (and document) `LOG_LEVEL` without prefix and `GOTRUE_LOG_FILE` with prefix, not?

**Comment by mraerino** [comment]
no, `LOG_FILE` does not work without prefix

**Comment by leomp12** [comment]
yup, and that's why I think current reference is a little bit confusing :stuck_out_tongue:

File: README.md [hunk 1]
```diff
@@ -75,7 +75,8 @@ Adds a prefix to all table names.
 ### Logging
 
 ```
-LOG_LEVEL=debug
+GOTRUE_LOG_LEVEL=debug
```
Comment (mraerino) [inline README.md]:
```suggestion
LOG_LEVEL=debug # available without GOTRUE prefix (exception)
```

**Pull request merged**